### DISCUSSION TITLE
Need some more info on changing the path variable

### DISCUSSION THREAD

**Discussion by eno812** [kudrykv/alfred-craftdocs#11] [General]
Is it possible to be a bit more specific about how to do this?

**Comment by kudrykv** [comment]
A few statements that you need to get familiar with:
- `PATH` contains a list of directories, which processes (e.g., Alfred workflow) uses for lookup of binaries;
- the list is colon-separated;
- it's best if these directories do not contain space symbols.

So, for example, `PATH`'s value can be `/usr/local/bin:/usr/bin:/Users/user/something/else`.

Here is a short video on how to edit it for the workflow (video is clickable, opens in a new tab):
![Edit PATH example](https://user-images.githubusercontent.com/12697803/127969604-cb03cacb-ae89-44d1-b04d-5597695d9419.mp4)

As an alternative, I suggest you try another workflow of mine to search through Alfred: [SearchIndex](https://github.com/kudrykv/alfred-craftdocs-searchindex). The alternative should work times faster.

**Reply by acegao9527** [reply]
yours is more convenient
